docs/api/blueprint/order/orders.apib

```diff
@@ -793,3 +626,114 @@ Create paypal payment
             "payment_id": "example",
             "error": null
         }
+
+## Create Order [/v1/orders/create-order]
```

Comment:
This should be placed in create order (1st section) place

Reply:
Done.